apps/src/craft/designer/game/GameController.js

```diff
@@ -1339,7 +1344,71 @@ class GameController {
     }
     // check the final state to see if its solved
     if (this.levelModel.isSolved()) {
-      this.endLevel(true);
+      const player = this.levelModel.player;
+      if (this.checkHouseBuiltEndAnimation()) {
```

Comment:
Perhaps out of scope for this PR, but I wonder if we could come up with some better naming. `checkHouseBuiltEndAnimation` just says is this a houseBuild level, so it presumably it means "check to see if this is a level where we should show the house build end animation". I wonder if it would better just be called `houseBuildLevel()`.

I also wonder if this method should be something like `animateIfSolved()` instead of `checkSolution()`?

In any case, I understand that this is mostly moving existing code, so feel free to use your discretion on whether you want to spend much/any time trying to make it cleaner as well.